packages/manager/modules/pci/src/projects/project/serving/namespace/models/add/add.html

```diff
@@ -0,0 +1,138 @@
+
+    <oui-stepper data-on-finish="$ctrl.addModel()">
+
+        <oui-step-form
+            data-header="{{ :: 'pci_projects_project_serving_namespace_models_add_type' | translate }}"
+            data-editable="!$ctrl.isAdding"
+            data-loading="$ctrl.isAdding"
+        >
+            <oui-radio
+                data-ng-repeat="workflowTemplate in $ctrl.workflowTemplates track by $index"
+                data-model="$ctrl.model.workflowTemplate"
+                data-name="workflowTemplate"
+                data-value="workflowTemplate"
+                data-required>
+                <span
+                    data-translate="{{:: 'pci_projects_project_serving_namespace_models_add_type_' + workflowTemplate.toLowerCase() }}"></span>
+                <button
+                    type="button"
+                    class="oui-popover-button"
+                    data-oui-popover="{{:: 'pci_projects_project_serving_namespace_models_add_type_' + workflowTemplate.toLowerCase() + '_popover' | translate }}"
+                    data-oui-popover-placement="auto"></button>
+            </oui-radio>
+        </oui-step-form>
+
+        <oui-step-form
+            data-header="{{ :: 'pci_projects_project_serving_namespace_models_add_model' | translate }}"
+            data-editable="!$ctrl.isAdding"
+            data-loading="$ctrl.isAdding">
+            <div class="container-fluid px-0" data-ng-if="$ctrl.model.workflowTemplate == $ctrl.PRESET_IMAGE">
+                <div class="row">
+                    <div class="col-md-6 col-xl-4 my-3"
+                         data-ng-repeat="preset in $ctrl.presetImages track by preset.id">
+                        <oui-select-picker
+                            data-name="{{:: preset.id }}"
+                            data-model="$ctrl.model.imageId"
+                            data-label="{{:: preset.description }}"
+                            data-variant="light"
+                            data-values="[preset]"
+                            data-required>
+                        </oui-select-picker>
+                    </div>
+                </div>
+            </div>
+
+            <div data-ng-if="$ctrl.model.workflowTemplate == $ctrl.BUILD_IMAGE">
+                <span data-translate="pci_projects_project_serving_namespace_models_add_storage_path_info"></span>
+
+                <oui-field data-size="xl">
+                    <oui-select
+                        disabled="$ctrl.paths.length == 0"
+                        id="storagePath"
+                        name="storagePath"
+                        model="$ctrl.model.storagePath"
+                        items="$ctrl.paths"
+                        data-required>
+                    </oui-select>
+                </oui-field>
+                <p data-ng-if="!$ctrl.paths.length">
+                    <span class="text-danger" ng-click="$ctrl.goToContainer()"
```

Comment:
No click on a `span`, `span`s are not `focusable`, please use a `button` instead